src/arload.py

```diff
@@ -123,6 +125,8 @@ def optarg(argflag, argv, pypname):
         if o in ('-h', '--help'): msgs.usage(None)
         elif o in ('-c', '--cpus'): argflag['run']['ncpus'] = a
         elif o in ('-v', '--verbose'): argflag['out']['verbose'] = int(a)
+    for a in arg:
```

Comment:
You should feed the flags first, like this:

pypit --use_masters --help lris_b_ls_600_4000_d560.red